packages/lesswrong/server/resolvers/conversationResolvers.ts

```diff
@@ -12,3 +43,78 @@ defineMutation({
     return true;
   }
 });
+
+//mutation to send one of a prespeified type of message to users
+defineMutation({
+  name: "sendEventTriggeredDM",
+  resultType: "Boolean!",
+  argTypes: "(eventType: String!)",
+  fn: async (_, {eventType}: {eventType: string}, context) => {
+
+    const { currentUser, Subscriptions, Conversations, Messages } = context;
+
+    if (!currentUser) {
+      throw new Error("You must be logged in to do this");
+    }
+
+    //check if eventType is one of dmTriggeringEvents, else return
+    if (!dmTriggeringEvents.has(eventType)) {
+      throw new Error("Invalid event type to trigger DM");
+    }
+
+    const lwAccount = await getAdminTeamAccount()
+
+    if (!lwAccount) {
+      throw new Error("Unable to find the lwAccount to send message to user")
+    }
+
+    if (eventType === "newFollowSubscription") {
+      
+      //check if this is a user's first follow
+      const numUsersFollows = await Subscriptions.find({
+        userId: currentUser._id,
+        type: "newActivityForFeed",
+        createdAt: {$gt: followSubscriptionStartDate}
+      }).count();
+
+      // if (numUsersFollows > 1) {
+      //   // already has followed and received a DM
+      //   return false;
+      // }
```

Comment:
cleanup

Reply:
or, rather, this doesn't seem like it should be commented out?

Reply:
Ah, yeah that was just for testing